src/index.html

```diff
@@ -13,7 +13,7 @@
   <section class="container">
     <h1>Leaderboard</h1>
     <div class="d-flex">
-      <div class="recent-scrore">
+      <div class="recent-score">
```

Comment:
- Kindly fix this issue related to your ``POST`` displayed in console:
![Screen Shot 2021-11-20 at 6 24 17](https://user-images.githubusercontent.com/22258274/142726202-3b1d3ec6-e50e-4399-9ebd-9499a1290247.png)

Reply:
@carlos-ssh 

Thanks for the review, I noticed that when I run the project on my computer, everything works perfectly without a single error in the console. I also took note that the URL displaying in your browser is `localhost:8080` while mine is being run on `localhost:8083`, which could potentially mean you're not running the correct project. Please make sure that they are no other projects running on your computer. I'll wait for your feedback, please.

Reply:
> * Kindly fix this issue related to your `POST` displayed in console:
>   ![Screen Shot 2021-11-20 at 6 24 17](https://user-images.githubusercontent.com/22258274/142726202-3b1d3ec6-e50e-4399-9ebd-9499a1290247.png)

Hi @carlos-ssh ,

This error was a result of your internet connection failing to resolve the API endpoint.

Have you considered running the project again, with a stable connection?